packages/okta-angular/README.md

```diff
@@ -231,26 +242,26 @@ this.oktaAuth.loginRedirect('/profile', {
 
 #### `oktaAuth.isAuthenticated()`
 
-Returns `true` if there is a valid access token or ID token.
+Returns a promise that returns `true` if there is a valid access token and ID token.
```

Comment:
This needs to be changed back to an `or`.

We should be gating authentication based on the presence of an `idToken` (first) or `accessToken`